services/web/server/src/simcore_service_webserver/login/handlers_change.py

```diff
@@ -65,10 +65,11 @@ async def reset_password(request: web.Request):
             )  # 401
 
     except web.HTTPError as err:
-        # Email wiht be an explanation and suggest alternative approaches or ways to contact support for help
+        # Email will be an explanation and suggest alternative approaches or ways to contact support for help
```

Comment:
not sure that comment still is correct since the email comes from the database right?